ide/libs.c.kohlschutter.junixsocket/external/junixsocket-common-2.2.0-license.txt

```diff
@@ -0,0 +1,210 @@
+Name: junixsocket
+License: Apache-2.0
+OSR: 4300
```

Comment:
This is an artifact from the oracle/sun era. Please remove.

Reply:
removed